webapp/src/main/webapp/WEB-INF/jsp/portlet/whoIsOnline.jsp

```diff
@@ -59,7 +59,11 @@
 <% } else { %>
   <div id="OnlinePortlet">
     <script type="text/javascript">
-      require(['SHARED/vue'], () => Vue.prototype.$updateApplicationVisibility(false, document.querySelector('#OnlinePortlet')));
+      document.onreadystatechange = () => {
+        if (document.readyState === 'complete') {
+          require(['SHARED/vue'], () => Vue.prototype.$updateApplicationVisibility(false, document.querySelector('#OnlinePortlet')));
```

Comment:
you can change the required module "SHARED/commonVueComponents" to ensure the init is made, rather than waiting on document init. In fact, in layout editor, the portlet won't show because the document has become ready already. Using the right dependant module will fix the behavior.

Reply:
ok, I will check this modification